src/ocm/components/clusterConfiguration/platformIntegration/PlatformIntegration.tsx

```diff
@@ -62,9 +62,13 @@ const PlatformIntegration = ({
       </SplitItem>
       <SplitItem>
         <PopoverIcon
-          bodyContent={messages[supportedPlatformIntegration]}
-          footerContent={
-            supportedPlatformIntegration === 'vsphere' && <PlatformIntegrationVsphere />
+          bodyContent={
+            <>
+              <Text>
+                <>{messages[supportedPlatformIntegration]}</>
+              </Text>
```

Comment:
Can't we do just this?
```suggestion
              <Text>{messages[supportedPlatformIntegration]}</Text>
```

Reply:
We can.